modules/ROOT/pages/dataweave-type-reuse-xml-schema.adoc

```diff
@@ -0,0 +1,429 @@
+
+=== Built-in XML Schema Types
+
+DataWeave supports the following XSD types where `xs` is the XMLSchema default namespace:
+
+[cols="2,2",options="header"]
+|===
+| XSD Type | Dataweave Type
+| `xs:number`| `Number`
+| `xs:int`| `Number`
+| `xs:integer`| `Number`
+| `xs:long`| `Number`
+| `xs:short`| `Number`
+| `xs:byte`| `Number`
+| `xs:double`| `Number`
+| `xs:decimal`| `Number`
+| `xs:unsignedLong`| `Number`
+| `xs:unsignedInt`| `Number`
+| `xs:unsignedShort`| `Number`
+| `xs:unsignedByte`| `Number`
+| `xs:positiveInteger`| `Number`
+| `xs:negativeInteger`| `Number`
+| `xs:nonNegativeInteger`| `Number`
+| `xs:nonPositiveInteger`| `Number`
+| `xs:hexBinary`| `Binary`
+| `xs:base64Binary`| `Binary`
+| `xs:boolean`| `Boolean`
+| `xs:any`| `String`
+| `xs:string`| `String`
+| `xs:normalizedString`| `String`
+| `xs:datetime`| `DateTime`
+| `xs:time`| `Time`
+| `xs:date`| `LocalDateTime`
+|===
+
+.XSD:
+[source,XML]
+----
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified"  xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:element name="Root">
+        <xs:complexType>
+            <xs:all>
+                <xs:element name="string" type="xs:string"/>
+                <xs:element name="normalizedString" type="xs:normalizedString"/>
+
+
+                <xs:element name="integer" type="xs:integer"/>
+                <xs:element name="negativeInteger" type="xs:negativeInteger"/>
+                <xs:element name="int" type="xs:int"/>
+                <xs:element name="unsignedInt" type="xs:unsignedInt"/>
+                <xs:element name="unsignedLong" type="xs:unsignedLong"/>
+                <xs:element name="unsignedShort" type="xs:unsignedShort"/>
+                <xs:element name="float" type="xs:float"/>
+                <xs:element name="double" type="xs:double"/>
+
+
+                <xs:element name="date" type="xs:date"/>
+                <xs:element name="dateTime" type="xs:dateTime"/>
+                <xs:element name="time" type="xs:time"/>
+
+
+                <xs:element name="boolean" type="xs:boolean"/>
+
+
+                <xs:element name="hexBinary" type="xs:hexBinary"/>
+                <xs:element name="base64Binary" type="xs:base64Binary"/>
+                <xs:element name="decimal" type="xs:decimal"/>
+
+
+
+
+
+
+
+
+            </xs:all>
+        </xs:complexType>
+    </xs:element>
+</xs:schema>
+----
+
+.Generated DataWeave Module:
+[source,dataweave,linenums]
+----
+%dw 2.0
+
+
+type RootElementDefinition = {| Root: { string: String, normalizedString: String, integer: Number, negativeInteger: Number, int: Number, unsignedInt: Number, unsignedLong: Number, unsignedShort: Number, float: Number, double: Number, date: LocalDateTime, dateTime: DateTime, time: Time, boolean: Boolean, hexBinary: Binary, base64Binary: Binary, decimal: Number } |}
+
+
+type Root = RootElementDefinition
+----
+
+== XSD Structure Mapping
+
```

Comment:
We should add a short intro here. 
@martincousido

Reply:
I would delete that title so the following titles are inside "From the XML Model to DataWeave"